### PR TITLE
Applied changes suggested by a Ruby linter

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Assorted changes include:
 - removal of unused variables
 - renaming of shadowing variables

@danmcp

### CODE REVIEW

File: test_pull_requests [hunk 1]
```diff
@@ -735,7 +735,7 @@ popd
 
     def get_trusted_trigger_time(pull_request, comments, settings)
       login = pull_request['user']['login']
-      updated_at, changed_after_eval = get_updated_at(pull_request, comments, settings)
+      updated_at = get_updated_at(pull_request, comments, settings)
```
Comment (danmcp) [inline test_pull_requests]: I think you want updated_at, _

File: test_pull_requests [hunk 2]
```diff
@@ -812,7 +812,7 @@ popd
           create_or_update_comment(pull_request['number'], repo, settings['test_prefix'], being_queued_comment, pull_request_comments)
         end
         pull_request_evaluated_time = get_evaluated_time(pull_request_comments, repo, settings)
-        pull_request_updated_at, pull_request_changed_after_eval = get_updated_at(pull_request, pull_request_comments, settings)
+        pull_request_changed_after_eval = get_updated_at(pull_request, pull_request_comments, settings)
```
Comment (danmcp) [inline test_pull_requests]: I think you want _, pull_request_changed_after_eval

File: test_pull_requests [hunk 3]
```diff
@@ -1268,7 +1268,7 @@ popd
             $stderr.puts "  Pull ##{req['number']} in repo '#{req['base']['repo']['name']}' is at build position ##{skipped_count_branch[settings['name']]}"
             # Get ahead of the game and pretest requests
             if settings['pretest_settings_key'] && settings['pretest_comment'] && settings['pretest_queue_threshold'] && (skipped_count_branch[settings['name']] >= settings['pretest_queue_threshold'])
-              trusted_trigger_time, trigger_login = get_trusted_trigger_time(req, comments, Properties['settings'][settings['pretest_settings_key']])
+              trusted_trigger_time = get_trusted_trigger_time(req, comments, Properties['settings'][settings['pretest_settings_key']])
```
Comment (danmcp) [inline test_pull_requests]: ditto

**Comment by stevekuznetsov** [comment]
@danmcp got rid of changes with the unused multiple returns ... didn't realize how much weird array unpacking was going on, and the `_` identifier would then similarly be flagged for being unused, right?

**Comment by stevekuznetsov** [comment]
@danmcp eh, maybe `_` is alright, even though it's a valid identifier at least it's more obvious the var is unused

File: test_pull_requests [hunk 4]
```diff
@@ -412,8 +412,8 @@ module Hub
 
       repo_to_pull_request.each do |repo, pull_request|
         if repo != base_repo
-          process_or_create_comment(pull_request['number'], repo, settings) do |comment_id, comment, comment_updated_at|
-            update_comment(comment_id, repo, running_comment)
+          process_or_create_comment(pull_request['number'], repo, settings) do |bot_comment_id, comment, comment_updated_at|
```
Comment (danmcp) [inline test_pull_requests]: where is bot_comment_id coming from here?

Comment (stevekuznetsov) [inline test_pull_requests]: Previously `comment_id` from `process_or_create_comment` was shadowing the outer `comment_id`, so I renamed the inner scope variable.

Comment (danmcp) [inline test_pull_requests]: Ah sorry, I misread and missed the do.  How about using c_id.  Since both are bot_comment_ids.

Comment (stevekuznetsov) [inline test_pull_requests]: I prefer the more verbose name, unless you have a reason you'd like the shorter one

Comment (danmcp) [inline test_pull_requests]: My objection is you are using the verbose name that doesn't separate it was the name above.  The less verbose name for the inner context is a common pattern.

**Comment by stevekuznetsov** [comment]
@danmcp updated name

**Pull request merged**